tests/integration/mixer/telemetry/metrics/istioctl_metrics_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"strconv"
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istioctl"
+)
+
+func TestIstioctlMetrics(t *testing.T) {
+	framework.
+		NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			label := "destination_service"
+			labelValue := "productpage.{{.TestNamespace}}.svc.cluster.local"
+			testMetric(t, ctx, label, labelValue)
+			workload := "productpage-v1"
+			testIstioctl(t, ctx, workload)
+		})
+}
+
+func testIstioctl(t *testing.T, ctx framework.TestContext, workload string) {
```

Comment:
question: should we test with `workloads []string` just to exercise watching multiple workloads with this command?